tests/browseruse/test_runner.py

```
@@ -32,18 +35,34 @@ async def run_all_tests():
+            traceback.print_exc()
             failed += 1
+            failed_tests.append(test_name)
         
         print(f"Test {test_name} completed.")
     
     print(f"\n{'='*60}")
     print(f"Test Summary: {passed} passed, {failed} failed")
+    if failed_tests:
+        print(f"Failed tests: {', '.join(failed_tests)}")
     print('='*60)
     
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)